Editor/CoreAPI/SettingsKeys.cs

```diff
@@ -14,6 +14,7 @@ public static class SettingsKeys
         public const string DeploymentBuildFolderPath = "DeploymentBuildFolderPath";
         public const string FleetName = "FleetName";
         public const string FleetId = "FleetId";
+        public const string FleetLocation = "FleetLocation";
```

Comment:
Remove? Now that we don't use this in the sample game

Reply:
Removed.